metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -30,20 +24,16 @@
         transaction_id INTEGER DEFAULT 0 NOT NULL,
         created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL"))
 
-(defmethod granule-column-sql true
+(defn granule-column-sql
```

Comment:
For easier readability, I would like to suggest we start naming our functions with more active verbs. 

'get-granule-columns-sql-query'

Reply:
create-granule-columns-sql

Reply:
I have updated the function names and used create-...

Reply:
+1